.travis.yml

```diff
@@ -7,8 +7,8 @@ env:
   matrix:
     - MOLECULE_DISTRO="centos:7"
     - MOLECULE_DISTRO="centos:8"
-    - MOLECULE_DISTRO="fedora:30"
     - MOLECULE_DISTRO="fedora:31"
+    - MOLECULE_DISTRO="fedora:32"
```

Comment:
please do this in a separate PR - unless this PR needs to specifically run the molecule tests on fedora 32

Reply:
Sure. Is it ok to remove fedora:30 in this PR? Otherwise, continuous-integration/travis-ci/pr does not pass...